aikau/src/main/resources/alfresco/documentlibrary/AlfBreadcrumb.js

```diff
@@ -102,7 +102,7 @@ define(["dojo/_base/declare",
       postCreate: function alfresco_documentlibrary_AlfBreadcrumbTrail__postCreate() {
          if (this.label)
          {
-            this.breadcrumbNode.innerHTML = this.encodeHTML(this.message(this.label));
+            this.breadcrumbNode.innerHTML = this.message(this.label);
```

Comment:
The filter publications are currently only issues by filters that have to be manually added to page, so there is minimal rish. The issue here was that when clicking on any of the tag filters we were getting double encoding of quotes (which is why I've removed the encoding).